src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -683,4 +704,149 @@ public function column_cb( $post ) {
 			<div class="locked-indicator"></div>
 		<?php
 	}
+
+	/**
+	 * Displays the pagination.
+	 *
+	 * @since 5.2.2
+	 * @access protected
+	 *
+	 * @param string $which
+	 */
+	protected function pagination( $which ) {
+		if ( empty( $this->_pagination_args ) ) {
```

Comment:
Make sure `$_pagination_args` is defined at class level looks like is only defined here. Additionally if this is going to be a private variable instead of using `_` prefix declare the access to the instance variable `private` and remove the prefix.

Reply:
It exists in the parent WP_List_Table class.

Reply:
Great wasn't aware this was a class extending from `WP_List_Table`